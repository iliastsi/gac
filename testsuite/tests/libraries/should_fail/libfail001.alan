output():proc
{
  writeInteger('t');
  writeInteger("123");

  writeByte(42);
  writeByte("123");

  writeChar(42);
  writeChar("123");

  writeString(42);
  writeString('t');
}
