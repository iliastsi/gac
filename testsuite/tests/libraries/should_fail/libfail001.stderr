libfail001.alan:3:16: Error: Type mismatch at `'t''
	Incompatible type of argument 1 of function `writeInteger'
	Expected `int' but argument is of type `byte'

libfail001.alan:4:16: Error: Type mismatch at `"123"'
	Incompatible type of argument 1 of function `writeInteger'
	Expected `int' but argument is of type `array of byte'

libfail001.alan:6:13: Error: Type mismatch at `42'
	Incompatible type of argument 1 of function `writeByte'
	Expected `byte' but argument is of type `int'

libfail001.alan:7:13: Error: Type mismatch at `"123"'
	Incompatible type of argument 1 of function `writeByte'
	Expected `byte' but argument is of type `array of byte'

libfail001.alan:9:13: Error: Type mismatch at `42'
	Incompatible type of argument 1 of function `writeChar'
	Expected `byte' but argument is of type `int'

libfail001.alan:10:13: Error: Type mismatch at `"123"'
	Incompatible type of argument 1 of function `writeChar'
	Expected `byte' but argument is of type `array of byte'

libfail001.alan:12:15: Error: Type mismatch at `42'
	Incompatible type of argument 1 of function `writeString'
	Expected `array of byte' but argument is of type `int'

libfail001.alan:13:15: Error: Type mismatch at `'t''
	Incompatible type of argument 1 of function `writeString'
	Expected `array of byte' but argument is of type `byte'

